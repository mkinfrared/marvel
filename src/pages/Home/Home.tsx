import React from "react";

import HeroesList from "containers/HeroesList";

const Home = () => {
  return (
    <div>
      <HeroesList />
    </div>
  );
};

export default React.memo(Home);
