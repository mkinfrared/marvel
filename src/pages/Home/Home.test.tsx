import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";

import Home from "./Home";

jest.mock("containers/HeroesList", () => () => "<HeroesList />");

describe("<Home />", () => {
  const Component = (
    <MemoryRouter>
      <Home />
    </MemoryRouter>
  );

  it("should be defined", () => {
    const { container } = render(Component);

    expect(container).toBeDefined();
  });

  it("match the snapshot", () => {
    const { container } = render(Component);

    expect(container).toMatchSnapshot();
  });
});
