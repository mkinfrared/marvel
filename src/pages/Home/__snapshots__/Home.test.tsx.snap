// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> match the snapshot 1`] = `
<div>
  <div>
    &lt;HeroesList /&gt;
  </div>
</div>
`;
